aws/cloud-formation/s3/update-s3.ps1

```diff
@@ -0,0 +1,3 @@
+aws cloudformation update-stack --stack-name digiroad-dev-api-s3 --capabilities CAPABILITY_NAMED_IAM --template-body file://aws/cloud-formation/s3/digiroad2-s3.yaml --parameters file://aws/cloud-formation/s3/DEV-s3-parameter.json
+
+aws cloudformation update-stack --stack-name qa-digiroad-api-s3 --capabilities CAPABILITY_NAMED_IAM --template-body file://aws/cloud-formation/s3/digiroad2-s3.yaml --parameters file://aws/cloud-formation/s3/QA-s3-parameter.json
```

Comment:
Onko tää stack name tarkoituksella näin päin qa:lla?

Reply:
Ihmettelin kanssa että miten ne on mennyt aikanaan noin ristiin, mutta joo, nimeäminen eroaa dev ja qa. Kyseessä on vanha stack ja stackin uudelleen nimeäminen ei ole mahdollista, joten vaihtoehdot olis joko poistaa kokonaan vanha stack ja luoda uusi oikealla nimellä tilalle tai sit annetaan noiden nimien olla.